### PR TITLE
Connect CDC Guide

### PR DESCRIPTION
**Pull request by kashifzeeshan**
Connect CDC Guide

### CODE REVIEW

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+1. Open Connect CDC Director.
+
+2. Click `New` option.
+
+3. Right click `Hosts, Servers, Tables` and select `New Host`.  
+
+4. Enter information for the `Host`, `Host` is the machine where Connect CDC Director is running.
+
+![Create New Host](Images/newhost.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:41]: Could we crop this image a bit more at the bottom to save space?

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+
+## Configure Postgres Distribution as Target Instance in Connect CDC Director
+
+The following steps demonstrate the setup and configuration of EDB Postgres Advanced Server as target instance to Connect CDC Director. 
+
+1. Right click `DBMS Servers` and `New Server`.
+
+2. Provide Connection Information for the EDB Postgres Advanced Server Target Instance.
+
+![Target Server](Images/targetserver.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:98]: Could we also crop this a bit more on the bottom?

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+
+The following steps demonstrate the creation of Distribution which will define how data will be replicated from source to target. In our case from EDB Postgres Advanced Server source to EDB Postgres Advanced Server target.
+
+1. Open Connect CDC Director.
+
+2. Right click on `Distributions` and select `New Distribution`.
+
+3. Enter Distribution information e.g `distribution name`, `description` etc and click ok.
+
+![New Distribution](Images/newdistribution.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:134]: Same for this image crop out the bottom some to help with white space.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+
+4. Go to Source DBMS `Server` —> `Tables` —> `Available Tables` —> `Public` (Schema name).
+
+5. Select the table(s) to replicate using the checking box in front of them and right click and select `Select for Distribution` —> `All Checked Tables`.
+
+![Select Tables for Replication](Images/selecttableforreplication.png)
+
+6. On the `Distributed Tables` Screen, select the `Target Server` under `Select one or more target servers` and select `Create tables on target` option and select the distribution under `Select a Distribution` option and click ok.
+
+![Distributed Tables](Images/distributetables.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:144]: Crop some of the white space out of the bottom.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+
+10. Once done, click `Finish`.
+
+11. To confirm the tables are created on `Target Server`, click the `Target DBMS Server` —> `Tables` —> `Receiving Tables`, the list of the tables will be displayed which are available on `Source DBMS Server`.
+
+![Confirm Tables Are Created on Target Server](Images/confirmtablesontarget.png)
+
+12. Open pgAdmin and connect to EDB Postgres Advanced Server Target Instance, and under Schema `public`, the tables will be created.
+
+![pgAdmin](Images/pgAdmin.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:164]: This is another really long image. Can we split this in half and do it side by side like the other pgAdmin image and have arrows pointing to where we need to go?

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 6]
```diff
@@ -0,0 +1,309 @@
...
+
+13. Mappings of tables setup for replication from EDB Postgres Advanced Server Source Instance to EDB Postgres Advanced Server Target Instance will be created under `Distributions` —> `Mappings`.
+
+![Mappings of Tables](Images/tablemappings.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:168]: For this image we need to crop out all of the white space at the bottom and I think highlight/have arrows pointing to the Distributions/Mappings portion at the bottom.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 7]
```diff
@@ -0,0 +1,309 @@
...
+
+2. Click `Model` —> `New Model` from Menu.
+
+3. Provide `Hostname/IP` and select the saved `Model` from the drop down list.
+
+![New Model](Images/newmodel.png)
+
+4. In the `Request` Section, select the `Distribution Model` that was created for copying data created in the above steps and right click and select `start`.
+
+![Distribution Model](Images/distributionmodel.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:208]: Can we crop out all of the extra space on this and just leave what we need to see if it won't warp?

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 8]
```diff
@@ -0,0 +1,309 @@
...
+
+5. Click on `Process` to see the progress.
+
+![Process](Images/Process.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:212]: This is another one where it comes across very small. If we really don't need it and can't make it bigger maybe delete this image.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 9]
```diff
@@ -0,0 +1,309 @@
...
+
+6. Open pgAdmin once the copy operation is completed successfully and connect the EDB Postgres Advanced Server Target Instance and check the tables and the data is copied to the tables.
+
+![pgAdmin](Images/pgAdmincopydone.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:216]: This is another image that comes over very small when looking at the page.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 10]
```diff
@@ -0,0 +1,309 @@
...
+
+Replication distribution will replicate all the changes done on the Source Tables to the Target Tables.
+
+1. Open Connect CDC MonCon.
+
+2. As the Model is updated in the Connect CDC Director to add the `Replication Distribution`, click on the `Model Update`, this will add the `Replication Distribution` to the Connect CDC MonCon Interface.
+
+3. The replication request will be displayed once the `Model` is updated successfully.
+
+![Replication Request](Images/replicationrequest3.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:252]: We can't really see anything in this image and the customers wouldn't be able to either. Do we need it? if not can it be deleted or is there something here we really want to show.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 11]
```diff
@@ -0,0 +1,309 @@
...
+
+4. In the `Request` Section, select the `Distribution Model` created for Replication and right click and select `Start`.
+
+![Distribution Model](Images/distrmodel.png)
+
+5. On the `Source Instance`, update the data to be replicated in the `Source Tables`.
+
+6. On the `Target Instance`, check data in the `Target Tables`, data will be updated there.
+
+![pgAdmin](Images/pgAdmintargettables.png)
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:262]: Is there anyway to make this bigger/more readable?

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 12]
```diff
@@ -0,0 +1,309 @@
...
+
+2. Go to `Source DBMS Server` —> `Tables` —> `Sending Tables`.
+
+3. Select the table(s) to perform data transformation using the checking box in front of them and right click and select `Properties` —> `Mappings`.
+
+4. On the `Mapping` tab, you define the column mappings for each of your target tables with respect to the source tables. The `Mapping` tab displays the corresponding columns in the `Target Column` and `Source Data` columns in each row. Connect CDC Director automatically mapped target columns to source columns that have the same name.
+
+When the Connect CDC Director maps a source table to a same-named target table, it also associates individual source table columns with same-named target table columns. You then select which of these column pairings you want to include in your data distribution, which additional column pairings you want to arrange, and what type of data transformation, if any, you want to assign to each column pairing.
+
+Mappings Tab has the following options
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:288]: For some reason the items under this are in code form. Like a block. Is that intentional?

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx [hunk 13]
```diff
@@ -0,0 +1,309 @@
...
+
+    - Receiving server list : The name of the target server is displayed.
+    - Receiving table list : The name of the target table is displayed. If it does not (for example, because its name does not match your source table’s), select the target table from the Receiving table list. The list includes all tables in the list of Available tables for the target server. Any table name with [M]preceding it is already mapped to a source table.
+    
+The Mapping tab grid displays each target table column and its datatype as well as its default corresponding source column, if any. Copy column under Method in the grid means the source column value is to be distributed without any special handling or transformation.
```
Comment (jkitchens32) [inline advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:293]: Same with these items. They are in like a block form.

**Review by jkitchens32 — CHANGES_REQUESTED** [review]
Review from conversation with Scott earlier on guide.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 1 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:41]: Done

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 2 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:98]: done

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 3 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:134]: done

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 4 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:144]: done

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 5 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:164]: done

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 6 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:168]: Image cropped but we don't need any arrow as its self explanatory to a the user who is following the steps.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 7 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:208]: done

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 8 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:212]: removed the image

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 9 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:216]: Nothing can be done for this, user can enlarge the page to see the details of the image.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 10 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:252]: deleted it.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 11 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:262]: updated the image and its the max we can do.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 12 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:288]: yes its intentional.

File: advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx (hunk 13 above)
Comment (kashifzeeshan) [reply advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx:293]: yes its intentional.

**Review by jkitchens32 — DISMISSED** [review]

**Review by jkitchens32 — APPROVED** [review]

**Pull request merged**